### ISSUE TITLE
Strip extra newline when reading `id_rsa.pub`

### ISSUE THREAD

**Issue by GregMefford** [nerves-project/nerves_firmware_ssh#7]
Reported by John Kaczmarek on Slack.

Doing `File.read!(Path.join(System.user_home!, ".ssh/id_rsa.pub"))` for authorized keys has an extra `\n` that needs to be stripped in order for it to work.

**Comment by axelclark** [comment]
Firmware updates seem to work for me even with with the `\n`, but here is a way to strip the new line:
```elixir
File.read!(Path.join(System.user_home!, ".ssh/id_rsa.pub"))
|> String.trim_trailing("\n")
```

I'm using the bash script on my MacBook Pro.

**Comment by JohnKacz** [comment]
Yea this needs more testing. I'll try to look into it soon and report back.

**Comment by axelclark** [comment]
@JohnKacz, I'm not sure what your setup is, but my fixes for issue #10 may help.

**Comment by fhunleth** [comment]
I don't think this is a problem (anymore?). If it shows up again, please re-open.
